exporter/azuremonitorexporter/README.md

```diff
@@ -19,6 +19,7 @@ The following settings can be optionally configured:
 - `endpoint` (default = `https://dc.services.visualstudio.com/v2/track`): The endpoint URL where data will be submitted.
 - `maxbatchsize` (default = 1024): The maximum number of telemetry items that can be submitted in each request. If this many items are buffered, the buffer will be flushed before `maxbatchinterval` expires.
 - `maxbatchinterval` (default = 10s): The maximum time to wait before sending a batch of telemetry.
+- `spaneventsenabled` (default = true): Enables export of span events.
```

Comment:
Will this be an unexpected change for users of this component?  Would it be better to default this to `false` for a couple releases, allowing time for users to opt in, and then eventually switch it to default `true` with a breaking change entry?

Reply:
Definitely.